schema.json

```diff
@@ -623,6 +576,12 @@
         "if": {
           "$ref": "#/definitions/if"
         },
+        "blocked_state": {
+          "type": "string",
+          "default": "passed",
```

Comment:
Based on empirical observation and conversation on Slack, it seemed that the default value of `blocked_state` for input steps was actually `failed` (as opposed to `passed` for block steps).

Reply:
```suggestion
        "blocked_state": {
          "type": "string",
          "default": "failed",
```

Reply:
I double checked in the backend code to be extra sure, it defaults to `null`,  but treats that as `passed`